.circleci/should_build_docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eo pipefail
+
+oldrev="origin/master"
+newrev="HEAD"
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PROJECT_DIR=$SCRIPT_DIR/..
+
+# Look for updated Dockerfiles in the PR
+cd "$PROJECT_DIR"
+COMMITS=$(git rev-list $oldrev..$newrev)
+DOCKER_FILES=$(
+  for commit in $COMMITS; do
+    git diff-tree --no-commit-id --name-only -r "$commit" -- "*.Dockerfile";
+  done
+)
+
+if [ "$DOCKER_FILES" ]; then
+  echo "Will run step."
```

Comment:
nit: maybe indicate the nature of the step ("rebuilding containers" or some such) in the message

Reply:
Ah, sounds good. Will patch it up.  I will put a README.md in the CI dir in later commit.

For background, CircleCI runs this script to determine it should continue to run the build step or terminate early.